### PR TITLE
Add note that 'tsh scp' always uses the SFTP protocol

### PR DESCRIPTION
**Pull request by capnspacehook**
There has been some confusion regarding the relationship between `tsh scp` and the SFTP protocol. Older versions of OpenSSH `scp` required users to pass the `-s` flag to enable use of the SFTP protocol, while `tsh scp` always uses the SFTP protocol.

### CODE REVIEW

File: docs/pages/connect-your-client/tsh.mdx [hunk 1]
```diff
@@ -599,6 +599,9 @@ scp"` and use the familiar syntax:
 $ scp -P 61122 -r files root@node:/path/to/dest
 ```
 
+Note that `tsh scp` always uses the SFTP protocol under the hood, no flags need
```
Comment (ptgott) [inline docs/pages/connect-your-client/tsh.mdx:602]:
```suggestion
Note that `tsh scp` always uses the SFTP protocol under the hood, so no flags need
```

File: docs/pages/connect-your-client/tsh.mdx [hunk 2]
```diff
@@ -586,7 +586,7 @@ $ tsh logout
 ## Copying files
 
 To securely copy files to and from cluster Nodes, use the `tsh scp` command. It
-is designed to mimic traditional `scp` as much as possible:
+is designed to mimic OpenSSH `scp` as much as possible:
```
Comment (ptgott) [inline docs/pages/connect-your-client/tsh.mdx:589]:
```suggestion
is designed to mimic OpenSSH's `scp` command as much as possible:
```

**Review by ptgott — APPROVED** [review]
Approved with minor suggestions

File: docs/pages/connect-your-client/tsh.mdx (hunk 1 above)
Comment (jakule) [reply docs/pages/connect-your-client/tsh.mdx:602]: I'd add that `tsh scp` uses SFTP starting from Teleport 11

File: docs/pages/connect-your-client/tsh.mdx (hunk 2 above)
Comment (jakule) [reply docs/pages/connect-your-client/tsh.mdx:589]: I'd also add that from Teleport 10, we support SFTP and SCP protocols, so `scp` and `sftp` tools from the OpenSSH package should work.

Comment (capnspacehook) [reply docs/pages/connect-your-client/tsh.mdx]: Done

File: docs/pages/connect-your-client/tsh.mdx [hunk 3]
```diff
@@ -599,6 +599,13 @@ scp"` and use the familiar syntax:
 $ scp -P 61122 -r files root@node:/path/to/dest
 ```
 
+Starting from Teleport 11 `tsh scp` always uses the SFTP protocol under
+the hood, so no flags need to be passed to enable this behavior.
+
+Starting from Teleport 10 the SCP and SFTP protocols are both supported by Nodes.
```
Comment (jakule) [inline docs/pages/connect-your-client/tsh.mdx:605]: 1. I talked with @alexfornuto as we think that "Server Access" sounds better
2. I forgot that we ported the SFTP server changes to v9 too.

```suggestion
Starting from Teleport 9 the SCP and SFTP protocols are both supported by Server Access.
```

**Review by jakule — APPROVED** [review]

**Pull request merged**